tooltester/src/main/java/io/dockstore/tooltester/runWorkflow/WorkflowRunner.java

```diff
@@ -83,13 +105,16 @@ public class WorkflowRunner {
      * @param pathOfTestParameter A relative path to the location of the test parameter (e.g. test/test-parameter-file.json)
      * @param extendedGa4GhApi
      */
-    public WorkflowRunner(String entry, String version, String pathOfTestParameter, ExtendedGa4GhApi extendedGa4GhApi, WorkflowsApi workflowsApi, String wdlConfigFilePath, String cwlConfigFilePath) {
+    @SuppressWarnings("checkstyle:parameternumber")
+    public WorkflowRunner(String entry, String version, String pathOfTestParameter, ExtendedGa4GhApi extendedGa4GhApi, WorkflowsApi workflowsApi, String wdlConfigFilePath, String cwlConfigFilePath, String clusterNameWDL, String clusterNameCWL) {
```

Comment:
No rush, but probably you're at the point where you can consider a builder design pattern.